app/controller/sdl/RCModulesController.js

```diff
@@ -44,6 +44,12 @@ SDL.RCModulesController = Em.Object.create({
       LIGHT: 'lightModels',
       HMI_SETTINGS: 'hmiSettingsModels'
     },
+    
+    /** 
+     * @description List of available UUIDs for SEAT module.
+     * @type {List}
+     */
+    seatUUIDs: [],
```

Comment:
@YarikMamykin why did you make this variable a class member? I see you are using it in one function only so it may be a local one

Reply:
@AKalinich-Luxoft removed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/f3589ff18a23e9a0c3ac58e006a8594b4f0576ba